netdaemon_alpha/config.json

```diff
@@ -25,12 +25,23 @@
   "options": {
     "log_level": "information",
     "app_config_folder": "/config/netdaemon4",
-    "app_assembly": ""
+    "app_assembly": "",
+    "mqtt_host": "",
```

Comment:
Are most people going to need these settings?
If not I would omit them from this and keep them in schema only

Reply:
That is a good point!